sessions/js-forms-2/simple-live-calculator/README.md

```diff
@@ -0,0 +1,19 @@
+# Simple Live Calculator
+
+Let's build a simple live calculator!
+
+## Task
+
+There are two form fields where the user can enter a value.
+As soon as one of these values change, both values shall be multiplied. The result is to be rendered with the help of the output element.
+
+## Hints
+
+The following hints might guide you:
+
+- You may need to declare global `let` variables for both input fields and initialize them with a starting number.
```

Comment:
I would argue that you don't need that variable so I would leave remove this hint.
```suggestion
```

Reply:
Agreed! I was too much captured by the prepared solution when I wrote these hints, but we should leave it more open for the students here.